### PR TITLE
Add 1.12 tests

### PR DESCRIPTION
**Pull request by wonjoolee95**
Add 1.12 tests

### CODE REVIEW

**Comment by JackCaoG** [comment]
I am assuming this is just copy from 1.11 and change it to 1.12? Is there a reason that it is still a draft? Do we need to dry run this before merging this?

**Comment by wonjoolee95** [comment]
> I am assuming this is just copy from 1.11 and change it to 1.12? Is there a reason that it is still a draft? Do we need to dry run this before merging this?

Yes, this is just a copy from 1.11 to 1.12 with some changes to 1.12 branch. We should be good to merge this, this won't affect any of the existing tests.

**Review by JackCaoG — APPROVED** [review]
Thanks! Let's try to get some run this weekend.

**Review by will-cromar — CHANGES_REQUESTED** [review]
I would copy from nightly to 1.12 instead of 1.11 to 1.12. 1.11 had quite a few broken test configs that have been fixed recently. Just remove the configs that shouldn't block the release (especially PjRt).

**Comment by wonjoolee95** [comment]
> I would copy from nightly to 1.12 instead of 1.11 to 1.12. 1.11 had quite a few broken test configs that have been fixed recently. Just remove the configs that shouldn't block the release (especially PjRt).

Thanks for pointing that out, @will-cromar. I updated the PR to copy from nightly tests instead. Also removed tests for `pjrt`. I'll follow-up with separate PRs on removing tests that we don't need for r1.12 release.

File: tests/pytorch/r1.12/common.libsonnet
Comment (will-cromar) [inline tests/pytorch/r1.12/common.libsonnet:45]: Do we need to install libtpu too?

File: tests/pytorch/r1.12/common.libsonnet
Comment (will-cromar) [inline tests/pytorch/r1.12/common.libsonnet:55]: 1.12

**Review by will-cromar — APPROVED** [review]
Thanks @wonjoolee95! Just a couple of comments

File: tests/pytorch/r1.12/common.libsonnet
Comment (wonjoolee95) [reply tests/pytorch/r1.12/common.libsonnet:45]: Absolutely, good catch. Added, `sudo pip3 install https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/wheels/libtpu-nightly/libtpu_nightly-0.1.dev20220518-py3-none-any.whl`.

**Comment by wonjoolee95** [comment]
Thanks! I don't seem to have merging rights. Please merge this when you get the time, thanks 😄, @will-cromar!

**Review by will-cromar — COMMENTED** [review]
I just checked the diff and there aren't any new files generated. You'll need to add 1.12 here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/pytorch/targets.jsonnet

File: tests/pytorch/targets.jsonnet
Comment (will-cromar) [inline tests/pytorch/targets.jsonnet:18]: r1_12

Comment (wonjoolee95) [reply tests/pytorch/targets.jsonnet]: Thanks lol, definitely not used to editing in vim

**Pull request merged**
